libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/scripts/modules/bit_esc.lua

```diff
@@ -90,6 +95,12 @@ function ESC:update()
             self.esc_rpm_nil_counter[i] = 0
             self.servo_out_nil_counter[i] = 0
         end
+        self.wait_for_safety_cooldown = now + self.DELAY_AFTER_SAFETY
+        return
+    end
+
+    -- Wait for a minute before checking for errors
```

Comment:
Tidy up this comment, but otherwise this is all good.

Reply:
Updated.